Resources/views/standard_layout.html.twig

```diff
@@ -261,7 +261,7 @@ file that was distributed with this source code.
 
                                                 {% if _actions|replace({ '<li>': '', '</li>': '' })|trim is not empty %}
                                                     <ul class="nav navbar-nav navbar-right">
-                                                    {% if _actions|split('<li>')|length > 2 %}
+                                                    {% if _actions matches '/(.*<li.*){2}/s' %}
```

Comment:
Why `.*` prefix?

And this regexp is too permissive.

Something like `<libutnotli>` will pass AFAIK.